test/controllers/SafeNavigationBaseController.test.ts

```diff
@@ -128,7 +128,7 @@ describe('Safe navigation base controller', () => {
             process.env.COOKIE_SECRET = 'super long and very secure secret';
 
             const sessionStore = createSubstituteOf<SessionStore>(substitute => {
-                substitute.store(Arg.any(), Arg.any(), Arg.any()).returns(EitherUtils.wrapValue('1'));
+                substitute.store(Arg.any(), Arg.any(), Arg.any());
```

Comment:
Did you forget to resolve?
```suggestion
                substitute.store(Arg.any(), Arg.any(), Arg.any()).resolve(1);
```